Dockerfile

```diff
@@ -14,9 +14,10 @@ RUN apt-get update -qq && apt-get -y --no-install-recommends install \
 WORKDIR primace
 COPY . .
 
-RUN Rscript -e 'install.packages("renv", repos = "https://rstudio.r-universe.dev")' 
+RUN Rscript -e 'install.packages("renv", repos = "https://rstudio.r-universe.dev")'
 RUN Rscript -e 'library(renv)' 
 RUN Rscript -e 'renv::restore()'
+RUN Rscript -e 'remotes::install_github("mlr-org/mlr3viz")'
```

Comment:
this causes installing version 0.6 which not compatible with mlr3proba 0.4.0. and cause dependecies conflict

Reply:
true, i would try to fix it